packages/templates/src/cli.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-await-in-loop, no-console */
```

Comment:
Add `packages/templates/src/cli.js` in the ESLint config here:

https://github.com/google/web-stories-wp/blob/9f4f35cf874b5671ac595f386eb3b8d21355f613/.eslintrc#L380-L392

and here:

https://github.com/google/web-stories-wp/blob/9f4f35cf874b5671ac595f386eb3b8d21355f613/.eslintrc#L393-L413